Claudia/src/main/java/com/endava/task/second/User.java

```diff
@@ -0,0 +1,47 @@
+package com.endava.task.second;
+
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+public class User {
+    public long id;
+    public String username;
+    public Map<String, Address> addressMap;
+
+    public User(String username, String addressType, Address address) {
+        this.id = Math.abs(new Random().nextLong());
+        this.username = username;
+        this.addressMap = addAddress(addressType, address);
+
+    }
+
+    public Map<String, Address> addAddress(String type, Address address) {
+        Map<String, Address> newAddress = new HashMap<>();
+        newAddress.put(type, address);
+        return newAddress;
+    }
+
+    public void printUser() {
+        System.out.println("User details are : ");
+        System.out.println("Id = " + this.id);
+        System.out.println("Username = " + this.username);
+        System.out.println("Addresses are:");
+        for (Map.Entry<String, Address> addressEntry : this.addressMap.entrySet()) {
+            System.out.println(addressEntry.getKey());
+            addressEntry.getValue().printAddress();
+        }
+    }
```

Comment:
Could you replace this with toString() method?

Reply:
Done.